.babelrc

```diff
@@ -1,3 +1,3 @@
-{
-  "presets": [ "es2015" ]
-}
+{
+  "presets": [ "es2015" ]
+}

```

Comment:
Hi,
I suspect that these files that you see as touched might do something with the line breaks or some issue with my GIT, as physically none of them, except directly related to the work done, was touched.